data/wanderers.txt

```diff
@@ -717,6 +737,9 @@ event "wanderers: tempest mass production"
 		remove variant
 			"Autumn Leaf"
 			"Summer Leaf"
+	fleet "Wanderer Miners"
+		add variant
+			"Tempest"
```

Comment:
Earth Shapers were added in 115e9df84808918adf12af471936c8c29a6b5337, well before mining was added ( f443ada0e1a4c ), so a rework of them to include a mining laser of sorts is plausible.

After I've finished that rework, I'll update the Wanderer Miners fleet to remove their warships. 
Thoughts on keeping the Summer Leaf involved in mining?